cli-output/src/cli_output.rs

```diff
@@ -3247,14 +3283,19 @@ mod tests {
             },
         ];
 
-        let c = CliVoteAccount {
+        let mut c = CliVoteAccount {
             account_balance: 10000,
             validator_identity: Pubkey::default().to_string(),
             epoch_rewards: Some(epoch_rewards),
             ..CliVoteAccount::default()
         };
         let s = format!("{c}");
-        assert!(!s.is_empty());
+        assert_eq!(s, "Account Balance: 0.00001 SOL\nValidator Identity: 11111111111111111111111111111111\nVote Authority: {}\nWithdraw Authority: \nCredits: 0\nCommission: 0%\nRoot Slot: ~\nRecent Timestamp: 1970-01-01T00:00:00Z from slot 0\nEpoch Rewards:\n  Epoch   Reward Slot  Time                        Amount              New Balance         Percent Change             APR  Commission\n  1       100          1970-01-01 01:00:00 +01:00  ◎0.000000010        ◎0.000000100               11.000%          10.00%          1%\n  2       200          1970-01-01 01:00:00 +01:00  ◎0.000000012        ◎0.000000100               11.000%          13.00%          1%\n");
```

Comment:
looks like these have an offset of +01:00 instead of +00:00 in the timestamps.

Reply:
hmm.. do different platforms print something different for unixtimestamp::default()? seems so..

Reply:
I don't think so; in `show_epoch_rewards` we explicitly have
```rust
                Local.timestamp_opt(reward.block_time, 0).unwrap(),
```

Reply:
Feels like that should always use UTC to be consistent with the other timestamps we display